File: exo6/exo6.py
#Ce code sert a repeter un mot ou une phrase que l'utilisateur a entrer.
#JesuScript
#beweb

quitter = False
oui = "o"
non = "n"
reponse = ""
flag = False

while quitter == False:
	print("Bienvenue chez le perroquet du capitaine")
	perroquet = raw_input("entrez une phrase ou un mot: ")
	print(str(perroquet))
	flag = True

	while flag == True:
		reponse = raw_input("Voulez vous quitter o/n ?")

		if oui == reponse.lower():
			quitter = True
			flag = False
		elif non == reponse.lower():
			print("non")
			flag = False
		else:
			print("Moi y a na pas comprendre ce que toi vouloir me dire")

File: exo6/exo6.php
<?php
//Ce code sert a repeter un mot ou une phrase que l'utilisateur a entrer.
//JesuScript
//beweb

$quitter = false;
$oui = "o"."\n";
$non = "n"."\n";
$reponse = "";
$flag = false;

while ($quitter == false) {
	echo("Bienvenue chez le perroquet du capitaine: "."\r"."\n");
	echo("Entrer une phrase ou un mot: ");
	$reponse = fgets(STDOUT);
	echo "$reponse";
	$flag = true;

	while ($flag == true){
		echo("Voulez vous quitter o/n ? "."\r"."\n");
		$reponse = fgets(STDOUT);
		if ($reponse == $oui) {
			$quitter = true;
			$flag = false;
		}elseif ($non == $reponse){
			$flag = false;
		}else{
			echo("Moi y a na pas comprendre ce que toi vouloir dire"."\r"."\n");
		}
	}
}
?>
